sunpy/tests/setup_command.py

```diff
@@ -109,7 +109,6 @@ def generate_testing_command(self):
                'online={online!r}, '
                'offline={offline!r}, '
                'figure={figure!r}, '
-               'coverage={1.coverage!r}, '
```

Comment:
How does this work? you are removing `coverage`... but then is used in the `if` in the file above..